subgraph.yaml

```diff
@@ -659,9 +659,9 @@ dataSources:
         - SaleAuction
       eventHandlers:
         - event: File(indexed bytes32,uint256)
-          handler: handleFile
+          handler: handleFile1
         - event: File(indexed bytes32,address)
-          handler: handleFile
+          handler: handleFile2
```

Comment:
Maybe change the name to handleFileAddress

Reply:
default function handle suites better